packages/extension/src/providers/common/ui/send-transaction/send-address-item.vue

```diff
@@ -7,7 +7,7 @@
       <img :src="network.identicon(network.displayAddress(account.address))" />
 
       <div class="send-address-item__name">
-        <h4>{{ account.name }}</h4>
+        <h4 v-if="account.name != null">{{ account.name }}</h4>
```

Comment:
should this be v-if="account.name", since `account.name `can also  be undefined?